drivers/hmis/app/graphql/types/hmis_schema/client.rb

```diff
@@ -43,6 +43,8 @@ def self.configuration
     hud_field :ethnicity, Types::HmisSchema::Enums::Hud::Ethnicity
     hud_field :veteran_status, Types::HmisSchema::Enums::Hud::NoYesReasonsForMissingData
     field :pronouns, [String], null: false
+    field :names, [HmisSchema::ClientName], null: false
+    field :primary_name, HmisSchema::ClientName, null: true
```

Comment:
Okay, I'll remove it. It was my guess at how we might want to access it if we weren't relying on the name fields on client, since that wasn't clear to me if we were going to continue to rely on those, but it we are then I agree this isn't necessary.